packages/expo-sensors/src/PlatformModule.ts

```diff
@@ -0,0 +1,19 @@
+import { EventEmitter } from 'expo-core';
+
+export default class PlatformModule {
```

Comment:
Not sure we need this base class - can you implement one of the native modules using the web APIs and see what that looks like? https://developers.google.com/web/fundamentals/native-hardware/device-orientation/